js/main.js

```diff
@@ -90,7 +90,8 @@ document.addEventListener('DOMContentLoaded', function () {
     var diagramType = selectDiagramElement.value
     var source = diagramSourceElement.value
     if (diagramType && source && source.trim() !== '') {
-      var urlPath = diagramType + '/svg/' + btoa(pako.deflate(textEncode(source), { level: 9, to: 'string' }))
+      var urlDiagramType = diagramType === 'c4plantuml' ? 'plantuml' : diagramType
```

Comment:
If I recall correctly, both URLs are working. Any reason why you want to use `plantuml` instead of `c4plantuml`?

Reply:
`Include` keywords such as !include <C4/C4_Context> are
available in plantuml-stdlib and not limited to C4.

example:`!include <awslib/AWSCommon>`

----
Using the limited URL c4plantuml may mislead users into believing that they cannot use stdlib other than c4plantuml.

Reply:
I see... I think we should discuss this change in https://github.com/yuzutech/kroki
Since C4 is now part of PlantUML, we might consider removing/deprecating the `c4plantuml` endpoint. As a first step, we could update the documentation to mention that it's recommended to use `plantuml` endpoint (and then we could indeed use `/plantuml` in the try section when C4 PlantUML is selected).